lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -46,7 +46,7 @@ namespace spv {
 
 typedef unsigned int Id;
 
-#define SPV_VERSION 10000
+#define SPV_VERSION 0x10000
 #define SPV_REVISION 2
```

Comment:
Couldn't `SPV_REVISION` and `Revision` be removed? They do not seem to be used anywhere in the code, and SPIRV-Tools did remove it in KhronosGroup/SPIRV-Tools#9.